chemprop/v2/featurizers/featurizers.py

```diff
@@ -24,12 +24,12 @@ def __len__(self) -> int:
 
 class BinaryFeaturizerMixin:
     def __call__(self, mol: Chem.Mol) -> np.ndarray:
-        return self.F.GetFingerprintAsNumPy(mol)
+        return self.F.GetFingerprintAsNumPy(mol).astype(np.int32)
```

Comment:
```suggestion
        return self.F.GetFingerprintAsNumPy(mol).astype(int)
```
use full precision for now as they'll get cast to half-precision later during batch collation